vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -535,6 +535,15 @@ bool FrameworkElementViewManager::UpdateProperty(
     } else if (propertyName == "accessibilityActions") {
       auto value = json_type_traits<winrt::IVector<winrt::react::uwp::AccessibilityAction>>::parseJson(propertyValue);
       DynamicAutomationProperties::SetAccessibilityActions(element, value);
+    } else if (propertyName == "display") {
+      if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::String) {
```

Comment:
I think you'll also want to handle propertyValue.IsNull().

Reply:
I think in that case we'd just want to default to visible, added that in :)

Reply:
I think you probably want to reset to default by calling ClearProperty(UIElement::VisibilityProperty()), but setting explicitly also works.

Reply:
I like that, that matches the pattern for other properties. Thanks!